genai_bench/cli/option_groups.py

```diff
@@ -611,6 +611,7 @@ def storage_auth_options(func):
     func = click.option(
         "--storage-bucket",
         type=str,
+        is_eager=True,
         help="Bucket/container name for storage provider.",
```

Comment:
what is the default of this parameter? if we set is_eager to True, and user doesn't pass in this parameter, it will give a default to this option. We need to double check its value